### ISSUE TITLE
domModuleAfterPolymer always fails with external script reference

### ISSUE THREAD

**Issue by JohnRiv** [PolymerLabs/polylint#52]
Currently, this setup will fail polylint with a `dom-module for dom-module-after-polymer should be a parent of it or earlier in the document.` error message even though it should pass:

**dom-module-after-polymer.html:**
```html
<link rel="import" href="../../../polymer/polymer.html">
<dom-module id="dom-module-after-polymer">
<template>
  I am an awesome element!
</template>
</dom-module>
<script src="dom-module-after-polymer.js"></script>
```

**dom-module-after-polymer.js:**
```js
Polymer({
  is: 'dom-module-after-polymer'
});
```

This is because the [scriptContent ](https://github.com/PolymerLabs/polylint/blob/885cc2ea9b9a3f1172a0a1862bdb946c523cb5bc/lib/linters.js#L441) of `<script src="dom-module-after-polymer.js"></script>` is empty, since it's just trying to grab the text content of that script node instead of actual contents of that external script, so it never finds the [otherDomModule](https://github.com/PolymerLabs/polylint/blob/885cc2ea9b9a3f1172a0a1862bdb946c523cb5bc/lib/linters.js#L444).

I'm not sure of the best way to fix this. If you could point me in the right direction I can work on a pull request.

**Comment by nomego** [comment]
According to polymer-project.org, the `script` tag should be within the `dom-module` tag?
Otherwise, this is a Hydrolysis issue reported at https://github.com/Polymer/hydrolysis/issues/185

**Comment by MeTaNoV** [comment]
From [Polymer documentation](https://www.polymer-project.org/1.0/docs/devguide/local-dom.html#template-stamping):
```
The <script> tag can be inside or outside of the <dom-module> element.
```

**Comment by nomego** [comment]
Ah, right.
All the examples had them within the `<dom-module>` though.
Anyway the hydrolysis issue should cover it.

**Comment by MeTaNoV** [comment]
definitely! :)
